apps/server/src/modules/deletion/entity/deletion-log.entity.ts

```diff
@@ -0,0 +1,67 @@
+import { Entity, Property } from '@mikro-orm/core';
+import { BaseEntityWithTimestamps, EntityId } from '@shared/domain';
+import { ObjectId } from '@mikro-orm/mongodb';
+import { DeletionDomainModel } from '../domain/types/deletion-domain-model.enum';
+import { DeletionOperationModel } from '../domain/types/deletion-operation-model.enum';
+
+export interface DeletionLogEntityProps {
+	id?: EntityId;
+	domain: DeletionDomainModel;
+	operation?: DeletionOperationModel;
+	modifiedCount?: number;
+	deletedCount?: number;
+	deletionRequestId?: ObjectId;
+	createdAt?: Date;
+	updatedAt?: Date;
+}
+
+@Entity({ tableName: 'deletionlogs' })
```

Comment:
The name can be singular as well as the collection is basically a complete `log` itself, but if you wanted to stick to the convention to use a plular form of the entity name then it's fine, let's leave it plural.